### PR TITLE
Created an API to retrieve LabeledInputView's value.

### LINKED ISSUE #116: Create an API for retrieving LabeledInputView's value
ATM, it's only possible via `view.element.value`.

Required by https://github.com/ckeditor/ckeditor5-link/issues/65.

### PR DESCRIPTION
**Pull request by oleq**
Closes #116.

### CODE REVIEW

File: src/inputtext/inputtextview.js [hunk 1]
```diff
@@ -59,4 +59,13 @@ export default class InputTextView extends View {
 	select() {
 		this.element.select();
 	}
+
+	/**
+	 * Returns the value of {@link ui.input.InputTextView#element}.
+	 *
+	 * @returns {*}
```
Comment (oskarwrobel) [inline src/inputtext/inputtextview.js:66]: I think that value of text input will always be a `string`

**Comment by oskarwrobel** [comment]
I have one doubt. Static getter is not observable.
Eg `button#isEnabled` can't be bound to `input#value` to disable submit when input is empty.

**Comment by oskarwrobel** [comment]
In f2f talk we have agreed to move binding problem to separate issue.
Besides question about docs everything works 👌

**Comment by oleq** [comment]
Moved to https://github.com/ckeditor/ckeditor5-ui/issues/110.

**Pull request closed without merging**